presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedSystemAccessControl.java

```diff
@@ -138,13 +141,20 @@ public SystemAccessControl create(Map<String, String> config)
         {
             requireNonNull(config, "config is null");
 
-            String configFileName = config.get(SECURITY_CONFIG_FILE);
+            Bootstrap bootstrap = new Bootstrap(
+                    binder -> configBinder(binder).bindConfig(FileBasedAccessControlConfig.class));
+            Injector injector = bootstrap.strictConfig()
+                    .doNotInitializeLogging()
+                    .setRequiredConfigurationProperties(config)
+                    .initialize();
+            FileBasedAccessControlConfig fileBasedAccessControlConfig = injector.getInstance(FileBasedAccessControlConfig.class);
+            String configFileName = fileBasedAccessControlConfig.getConfigFile();
```

Comment:
`getConfigFile` has `@NotNull` validation so the check below is not needed and can be removed.

Reply:
Hey @kokosing , I have done this change. can we merge this pull request?